frontend/views/components/Modal/Modal.vue

```diff
@@ -6,20 +6,18 @@
 <script>
 import sbp from '~/shared/sbp.js'
 import { OPEN_MODAL, REPLACE_MODAL, CLOSE_MODAL } from '@utils/events.js'
-import { logExceptNavigationDuplicated } from '@controller/utils/misc.js'
-
 export default {
   name: 'Modal',
   data () {
     return {
       content: null, // This is the main modal
       subcontent: [], // This is for collection of modal on top of modals
-      replacement: null
+      replacement: null // This let us replace the modal once the first one is close without updating the url
     }
   },
   created () {
     sbp('okTurtles.events/on', OPEN_MODAL, component => this.openModal(component))
-    sbp('okTurtles.events/on', CLOSE_MODAL, component => this.unloadModal(component))
+    sbp('okTurtles.events/on', CLOSE_MODAL, this.unloadModal)
```

Comment:
we can do the same in all 3 events: OPEN_MODAL, CLOSE_MODAL and REPLACE_MODAL, a direct callback